conduit/pipeline_test.go

```diff
@@ -327,47 +339,29 @@ func TestPipelineCpuPidFiles(t *testing.T) {
 		round:        0,
 	}
 
-	var wg sync.WaitGroup
-
-	wg.Add(1)
-
-	go func() {
-		err := pImpl.Start()
-		assert.Nil(t, err)
-		wg.Done()
-	}()
-
-	// Give some time for the goroutine to start
-	time.Sleep(1 * time.Second)
```

Comment:
Why isn't the delay needed anymore?

Reply:
+1...just curious

Reply:
`Pipeline.Start` synchronously creates the PID file and CPU Profile if necessary, launches the goroutine which runs block imports, and then returns. So since we're not creating these files in a goroutine anymore they will always be created by the time we've returned.